docs/Changelog.md

```diff
@@ -39,6 +39,7 @@
   * Added authorization check for listing activities and downloading attachments in scenario activity API
   * Fixed the ability to download an unrelated attachment from a given scenario
 * [#5522](https://github.com/TouK/nussknacker/pull/5522) Scenario status caching more often
+* [#5519](https://github.com/TouK/nussknacker/pull/5519) Improvements: Flink's REST API caching
```

Comment:
Let's bundle both entries into one. From user perspective it is a one subject around caching of statuses